src/components/header/Header.tsx

```diff
@@ -0,0 +1,89 @@
+/// <reference types="vite-plugin-svgr/client" />
+import React, { Suspense, useCallback } from "react";
+import AppIcon from "@/assets/app-icon.svg?react";
+import ArrowDown from "@/assets/arrow-down.svg?react";
+import Typography from "@/components/typography/Typography";
+import Breadcrumb, { BreadcrumbItem } from "./breadcrumb/Breadcrumb";
+import { APP_NAME } from "@/common/consts";
+import { useRouter } from "@tanstack/react-router";
+import { useAuth } from "@/hooks/useAuth";
+import { lazy } from "react";
+
+type HeaderProps = {
+  breadcrumbs: BreadcrumbItem[];
+  user: string;
+};
+
+function DynamicLoader(thumbnail: number) {
```

Comment:
Cool generic component that should (I believe) be outside of the `Header` componet